test/test_torch.py

```diff
@@ -2638,6 +2638,18 @@ def assertEqual(device_str, fn):
                 if TEST_NUMPY:
                     assertEqual('cuda:1', lambda: torch.tensor(np.random.randn(2, 3), device='cuda:1'))
 
+    # def test_qtensor(self):
+    #     num_elements = 10
+    #     r = torch.ones(num_elements, dtype=torch.float)
+    #     scale = 1.0
+    #     zero_point = 2
+    #     qr = r.quantize_linear(scale, zero_point)
```

Comment:
To make this work, we need to add `requires_tensor: True` to the new functions in native_functions.yaml, so that the Variable `self` will be unpacked and non-VariableType dispatch is called in VariableType.cpp's `quantize_linear()` function.